static/js/tab_bar.js

```diff
@@ -51,9 +51,7 @@ exports.colorize_tab_bar = function () {
     const filter = narrow_state.filter();
     if (filter === undefined || !filter.has_operator('stream') || !filter._stream_params) {return;}
     const color_for_stream = stream_data.get_color(filter._stream_params._stream_name);
-    $("#tab_list .fa-hashtag").css('color', color_for_stream);
-    $("#tab_list .fa-lock").css('color', color_for_stream);
-    $("#tab_list .fa-globe").css('color', color_for_stream);
+    $("#tab_list .fa-" + filter.get_icon()).css('color', color_for_stream);
```

Comment:
This would be much better as `$(#tab_list .navbar_stream_icon).css(...)` (or whatever the right name is), so that we're updating based on the logical purpose of the icon, not whatever image it happens to be.

Reply:
I've currently changed this to `$("#tab_list .stream  > .fa")`... part of the reason I'm avoiding a new class is because I'm hoping to extract the icon and title into a separate, simple template (see https://github.com/zulip/zulip/pull/15335/commits/0a790a8a89db56a7fa1430a876e0de48d6aaf02c)

but if `$("#tab_list .stream  > .fa")` isn't good enough, I can make the change

Reply:
I think having an class naming the role of this icon would be better.

As a sidenote, we should rename that `.stream`; we avoid super generic class names like that unless it's a reusable component, which this certainly is not.